tool/gravity/cli/plan.go

```diff
@@ -64,9 +64,15 @@ func initUpdateOperationPlan(localEnv, updateEnv *localenv.LocalEnvironment) err
 	return trace.Wrap(err)
 }
 
-func displayOperationPlan(localEnv, updateEnv, joinEnv *localenv.LocalEnvironment, operationID string, format constants.Format) error {
-	operations, err := getLastOperation(localEnv, updateEnv, joinEnv, operationID)
+func displayOperationPlan(localEnv *localenv.LocalEnvironment, environ LocalEnvironmentFactory, operationID string, format constants.Format) error {
+	operations, err := getLastOperation(localEnv, environ, operationID)
 	if err != nil {
+		if trace.IsNotFound(err) {
+			return trace.NotFound(`no operation found.
+This usually means that the installation has failed to start or was not started.
```

Comment:
Nit: Can this also mean that "join" operation has failed to start or was not started?

Reply:
Possible - I'll update the message.